sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableService.java

```diff
@@ -30,6 +29,7 @@
 import java.util.concurrent.CompletionStage;
 import org.apache.beam.sdk.io.gcp.bigtable.BigtableIO.BigtableSource;
 import org.apache.beam.sdk.values.KV;
+import org.threeten.bp.Duration;
```

Comment:
I think we want to use java.time for beam...threetenbp is a legacy thing in veneer